### PR TITLE
fix: re-enable & update content as per BCS

### PR DESCRIPTION
**Pull request by virajvchaudhari**
*Issue #, if available:*

*Description of changes:*
- Re-enabling Braket checkstyle checks and fix the existing issues. 

*Testing done:*
tox 

## Merge Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your pull request._

#### General

- [ ] I have read the [CONTRIBUTING](https://github.com/aws/amazon-braket-sdk-python/blob/main/CONTRIBUTING.md) doc
- [ ] I used the commit message format described in [CONTRIBUTING](https://github.com/aws/amazon-braket-sdk-python/blob/main/CONTRIBUTING.md#commit-your-change)
- [ ] I have updated any necessary documentation, including [READMEs](https://github.com/aws/amazon-braket-sdk-python/blob/main/README.md) and [API docs](https://github.com/aws/amazon-braket-sdk-python/blob/main/CONTRIBUTING.md#documentation-guidelines) (if appropriate)

#### Tests

- [ ] I have added tests that prove my fix is effective or that my feature works (if appropriate)
- [ ] I have checked that my tests are not configured for a specific region or account (if appropriate)

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by speller26 — APPROVED** [review]
Will we be adding this for our other repositories too?

**Comment by virajvchaudhari** [comment]
> Will we be adding this for our other repositories too?

Yes, have added it as a backlog item.

**Comment by virajvchaudhari** [comment]
Don't merge this yet: There are few inconsistencies in the docs that needs a check.

File: src/braket/ahs/shifting_field.py [hunk 1]
```diff
@@ -95,11 +95,11 @@ def stitch(
...
-            * "mean" - take the average of the boundary value points of the first
-              and the second time series.
-            * "left" - use the last value from the left time series as the boundary point.
-            * "right" - use the first value from the right time series as the boundary
-              point.
+                * "mean" - take the average of the boundary value points of the first
+                and the second time series.
+                * "left" - use the last value from the left time series as the boundary point.
+                * "right" - use the first value from the right time series as the boundary
+                point.
```
[the remark this answers is not in the document]
Comment (virajvchaudhari) [reply src/braket/ahs/shifting_field.py]: Fixed.

File: src/braket/ahs/driving_field.py [hunk 2]
```diff
@@ -95,11 +95,11 @@ def stitch(
             other (DrivingField): The second shifting field to be stitched with.
             boundary (StitchBoundaryCondition): {"mean", "left", "right"}. Boundary point handler.
                 Possible options are
-            * "mean" - take the average of the boundary value points of the first
-              and the second time series.
-            * "left" - use the last value from the left time series as the boundary point.
-            * "right" - use the first value from the right time series as the boundary
-              point.
+                * "mean" - take the average of the boundary value points of the first
```
[the remark this answers is not in the document]
Comment (virajvchaudhari) [reply src/braket/ahs/driving_field.py:98]: Fixed.

Comment (krneta) [inline src/braket/ahs/driving_field.py:98]: Did you check the output of this? If not, can you convert the '*' to '-' and indent by 2?

File: src/braket/ahs/shifting_field.py [hunk 3]
```diff
@@ -94,15 +94,44 @@ def stitch(
...
             boundary (StitchBoundaryCondition): {"mean", "left", "right"}. Boundary point handler.
+
                 Possible options are
-            * "mean" - take the average of the boundary value points of the first
-              and the second time series.
-            * "left" - use the last value from the left time series as the boundary point.
-            * "right" - use the first value from the right time series as the boundary
-              point.
+
+              - "mean" - take the average of the boundary value points of the first
```
Comment (krneta) [inline src/braket/ahs/shifting_field.py:100]: I think you may need to indent this to work properly.

File: src/braket/aws/aws_session.py [hunk 4]
```diff
@@ -211,8 +211,7 @@ def create_quantum_task(self, **boto3_kwargs) -> str:
         Create a quantum task.
 
         Args:
-            ``**boto3_kwargs``: Keyword arguments for the Amazon Braket `CreateQuantumTask`
-              operation.
+            `**boto3_kwargs`: Keyword arguments for the Amazon Braket `CreateQuantumTask` operation.
```
Comment (krneta) [inline src/braket/aws/aws_session.py:214]: You can leave the double `` here and below.

File: src/braket/timings/time_series.py [hunk 5]
```diff
@@ -186,45 +185,18 @@ def stitch(
...
+            boundary (StitchBoundaryCondition): {"mean", "left", "right"}. Boundary point handler.
+
+                Possible options are
 
-                - "mean" - take the average of the boundary value points of the first
-                  and the second time series.
-                - "left" - use the last value from the left time series as the boundary point.
-                - "right" - use the first value from the right time series as the boundary
-                  point.
+              - "mean" - take the average of the boundary value points of the first
```
Comment (krneta) [inline src/braket/timings/time_series.py:192]: I think you need to indent this by 4 if you want it to look right.

Comment (virajvchaudhari) [reply src/braket/timings/time_series.py:191]: If we indent by 4, half the text is rendered as bold.

File: src/braket/aws/aws_session.py (hunk 4 above)
Comment (virajvchaudhari) [reply src/braket/aws/aws_session.py]: Fixed, thanks!

File: src/braket/ahs/shifting_field.py (hunk 3 above)
Comment (virajvchaudhari) [reply src/braket/ahs/shifting_field.py:99]: Same as below, indent by 4 causes the rendering issue of making half the sentence as bold.

File: src/braket/ahs/driving_field.py (hunk 2 above)
Comment (virajvchaudhari) [reply src/braket/ahs/driving_field.py]: Missed out checking this. Fixed it by converting to `-`, Thanks!

**Review by krneta — APPROVED** [review]

**Review by krneta — APPROVED** [review]

**Pull request merged**